### DISCUSSION TITLE
Entries in the collection - XML Format

### DISCUSSION THREAD

**Discussion by mrdavet247** [dylanljones/pyrekordbox#117] [Q&A] [answered]
Firstly, thanks for creating this Library, it's great 👍

Is it possible to fetch the number of Entries in the Collection when using the XML format?

I've done this myself using the standand Python XML library. It means I have to parse the XML file myself, so twice, just to get this value.

**Comment by dylanljones** [comment, marked as answer]
Hey,

yes, that's a simple property:

````python
from pyrekordbox.rbxml import RekordboxXml

xml = RekordboxXml("database.xml")
n = xml.num_tracks
print(f"Number of tracks: {n}")
````

**Reply by mrdavet247** [reply]
Thanks for getting back to me. It's my first Python program so I'm still learning...

**Reply by dylanljones** [reply]
No worries, glad i could help:)
